apps/console/src/features/organizations/pages/organization-edit.tsx

```diff
@@ -53,6 +59,20 @@ const OrganizationEditPage: FunctionComponent<OrganizationEditPagePropsInterface
     const [ isAuthorizedOrganization, setIsAuthorizedOrganization ] = useState(false);
     const [ filterQuery, setFilterQuery ] = useState<string>("");
 
+    const { switchOrganization, switchOrganizationInLegacyMode } = useOrganizationSwitch();
+    const { legacyAuthzRuntime }  = useAuthorization();
+    const { onSignIn } = useSignIn();
+    const { isFirstLevelOrganization, isSuperOrganization } = useGetCurrentOrganizationType();
+
+    const shouldSendRequest: boolean = useMemo(() => {
+        return (
+            isFirstLevelOrganization() || isSuperOrganization() || window[ "AppUtils" ].getConfig().organizationName
```

Comment:
Do we need the org name in this check here, as the first org check is already done by `isFirstLevelOrganization`?

```suggestion
            isFirstLevelOrganization() || isSuperOrganization()
```